templates/preview-a4a.html

```diff
@@ -144,16 +151,30 @@ <h1 id="abe-preview-header-title" class="h3 center truncate caps">{{subHeading}}
                 layout="{{layout}}"
                 data-slot="{{metadata.adSlot}}"></amp-ad>
         {{/metadata.adSlot}}
-        {{^metadata.adSlot}}
-        <amp-ad layout="{{layout}}"
-          width="{{width}}"
-          height="{{height}}"
-          type="fake"
-          id="i-amphtml-demo-fake"
-          a4a-conversion="true"
-          disable3pfallback="false"
-          src="{{a4aEmbedUrl}}"></amp-ad>
-        {{/metadata.adSlot}}
+        {{#metadata.adExp}}
+        <amp-ad width="{{width}}"
+                height="{{height}}"
+                type="fake"
+                data-use-a4a="true"
+                force3p="false"
+                fakesig="true"
+                src="{{a4aEmbedUrl}}"
+                disable3pfallback="false">
+        <div placeholder="" class="amp-hidden"></div>
```

Comment:
For some reason, the ad got default placeholder on top before timeout. This is a temporary hack to remove that. Since the whole PR is temporary, I think it's OK to live with this at now.